### ISSUE TITLE
Doctests are used incorrectly

### ISSUE THREAD

**Issue by Maximilian-Stefan-Ernst** [formal-methods-mpi/Taxonomy.jl#90]
For example, for `CFA` and `Factor` the doctests are used to verify that the constructors work as intended.

**Comment by aaronpeikert** [comment]
That is true (at least with the new print methods). We should test constructors and *all* extractors for each taxon.
